.github/workflows/fbgemmci.yml

```diff
@@ -61,8 +61,8 @@ jobs:
         cmake -DFBGEMM_LIBRARY_TYPE=shared ..
         make
 
-    - name: Test shared FBGEMM  lib
-      if: contains(runner.os, 'Linux')   # not run on macos-latest now due to supporting AVX2
+    - name: Test shared FBGEMM lib
+      if: contains(runner.os, 'linux')   # not run on macos-latest now due to supporting AVX2
```

Comment:
For the comment: does it mean "we cannot run on MacOS-latest, because it doesn't have support for AVX2"?

Reply:
Yes.